### PR TITLE
Gpr 2023.03.04

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_5.txt [hunk 1]
```diff
@@ -489,7 +490,7 @@ But I can't put up with any more of this vampire's ranting.
 [sha:72c321d9214e58962e8996d52051045466745837]{
 -- Page 546, Offset 34839.
 -- アイツは吸血鬼です、周りをよく見てください！」
-She's a vampire! Take a good look around you!"
+She's a vampire! I mean just look around you!"
```
Comment (yk1512) [inline script/Ciel/Day 13/13_01_01_CIEL13_5.txt:493]: 'I mean' is redundant imo

File: script/Ciel/Day 13/13_01_01_CIEL13_5.txt [hunk 2]
```diff
@@ -564,12 +565,12 @@ You're pissed off, are you?
 -- ……だいたいアンタの目的は俺だろう。ロアを殺したいんなら俺を狙え。吸血鬼に成り下がったアンタに、先輩を責める資格はない」
 Besides, you're after %{i}me%{/i}, aren't you?
  If you want to kill Roa, then here I am.
- You have no right to condemn Senpai after debasing yourself by becoming a vampire!"
+ You have no right to condemn Senpai after having degraded into a vampire!"
```
Comment (yk1512) [inline script/Ciel/Day 13/13_01_01_CIEL13_5.txt:568]: Now that you've degraded into a vampire, you have no right to condemn Senpai!
// Original structure feels a little awkward to me, I think its because of the 'after having'

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 13/13_01_01_CIEL13_5.txt (hunk 1 above)
Comment (Avuxo) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:493]: I disagree but think there should be a comma after "mean"

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 13/13_01_01_CIEL13_5.txt (hunk 2 above)
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:568]: Have a suggestion for the follow up?

File: script/QA/QA_18.txt [hunk 3]
```diff
@@ -116,19 +116,20 @@ We do not have much time, so perhaps you%{n}
...
 // in the same vein as above
-Hmm, but I get the sense that if I mess%{n}
+Hmm, but I get the sense that if I'm not%{n}
 }
 [sha:ccde253fa43353b0a8077f46c90a78a3162015f3]{
 -- Page 123, Offset 43544.
 -- この世から消滅しそうなエクサトン。
-with her, an exaton of matter will be%{n}
+careful with her, an exaton of matter will be%{n}
 annihilated...
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/QA/QA_18.txt:133]: Doesn't sound exactly right to me - the matter either annihilates something or is annihilated by something, and the change kinda makes it sound more like it happens in a vacuum, rather than it being annihilated _by neco arc_ (dark matter)

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/QA/QA_18.txt:133]: I thought the anti-matter type of annihilation was intransitive

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/QA/QA_18.txt:133]: Maybe? It just feels like an awkward construction, since the usual usage of annihilate _is_ transitive, and I worry it'll get misread. Maybe reword it? 
> ...not careful with her, it'll result in the annihilation of an exaton of matter.
